V3-Upgrade-Guide.md

```diff
@@ -60,6 +60,8 @@ The following sub-sections describe affected modules, functions and interfaces w
 ### Package `@sap-cloud-sdk/connectivity` <!-- omit from toc -->
 
 - The generic type parameter `JwtKeysT` in `JwtKeyMapping` is now narrowed to extend `string`.
+- The property `url` on the `Destination` interface becomes optional.
+It is only present for HTTP destinations and not for Mail destinations.
```

Comment:
```suggestion
  It is only present for HTTP destinations and not for Mail destinations.
```
[q] I assume this is meant to be part of the bullet point. Without the leading spaces it would render as a line below the list.